bin/blacklist.json

```diff
@@ -0,0 +1,7 @@
+{
+  "pipelines": [
+    "atacseq",
+    "ampliseq",
+    "hlatyping"
```

Comment:
Are these not the pipelines that _should_  be synced? I was more expecting this file to list all of the repos that shouldn't be synced (eg. `tools`, `nf-co.re` etc). Then we don't have to update it for every new pipeline.